compiler/vm/core/src/shadowframes.c

```diff
@@ -1,6 +1,11 @@
 #include <robovm.h>
 
-void rvmPushShadowFrame(Env* env, ShadowFrame* frame) {
+#define LOG_TAG "shadowframes"
+
+void rvmPushShadowFrame(Env* env, void* functionAddress) {
+    ShadowFrame* frame = rvmAllocateMemory(env, sizeof(ShadowFrame));
```

Comment:
Could raw malloc be used? I thought I need to call rvmAllocateMemory for the memory to be collectable by the GC.